src/Microsoft.Extensions.WebEncoders/EncoderServiceCollectionExtensions.cs

```diff
@@ -17,14 +20,23 @@ public static class EncoderServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/>.</param>
         /// <returns>The <see cref="IServiceCollection"/> instance after the encoders have been added.</returns>
-        public static IServiceCollection AddWebEncoders(this IServiceCollection services)
+        public static void AddWebEncoders(this IServiceCollection services)
```

Comment:
I was thinking you meant to have a common interface for all builders across the stack. If you just mean builder implementing IServiceCollection that could work, but I don't see how that would help maintain the chaining behavior.

Chaining breaks because some methods now return void instead of the IServiceCollection and that change we want to keep. With the current builder you can just do .Services and keep chaining things if you choose to do so.